tests/integration/pkg/tests/fleet/fleetcluster_test.go

```diff
@@ -0,0 +1,110 @@
+package fleetcluster_test
+
+import (
+	"testing"
+	"time"
+
+	fleetv1api "github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+	mgmt "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	provisioningv1api "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	"github.com/rancher/rancher/tests/integration/pkg/clients"
+	"github.com/rancher/rancher/tests/integration/pkg/cluster"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	waitFor = 5 * time.Minute
+	tick    = 2 * time.Second
+)
+
+func TestFleetCluster(t *testing.T) {
```

Comment:
Would you mind moving this to the new location i.e. `tests/v2prov/pkg/tests` and renaming the test so that it runs in CI?

https://github.com/rancher/rancher/tree/release/v2.7/tests/v2prov#general for more context on the new location

Reply:
If this is merged as it stands it won't be run by anything.

Reply:
Indeed, I think it will also conflict with the agent customization test